src/app/organizations/tools/exposed-passwords-report.component.ts

```diff
@@ -1,17 +1,12 @@
-import {
-    Component,
-    ComponentFactoryResolver,
-} from '@angular/core';
+import { Component, ComponentFactoryResolver } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 
 import { AuditService } from 'jslib/abstractions/audit.service';
 import { CipherService } from 'jslib/abstractions/cipher.service';
 import { MessagingService } from 'jslib/abstractions/messaging.service';
 import { UserService } from 'jslib/abstractions/user.service';
 
-import {
-    ExposedPasswordsReportComponent as BaseExposedPasswordsReportComponent,
-} from '../../tools/exposed-passwords-report.component';
+import { ExposedPasswordsReportComponent as BaseExposedPasswordsReportComponent } from '../../tools/exposed-passwords-report.component';
```

Comment:
What about these formatting cases? This is breaking the 120 max line rule... ?

Reply:
Prettier has no explicit max line rule. It will make a best effort to strive to use a specific line length but makes no promise that it will do so if the alternative is sub-optimal.